docs/migration-guide-2.3-2.4.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Migration Guide 2.3-2.4
+---
+
+In Kyma 2.0, we introduced a [new way to reach registered services and some Application Connectivity improvements](https://kyma-project.io/blog/2021/12/7/release-notes-20/#application-connectivity), which were related to [Service Catalog removal](https://kyma-project.io/blog/2021/12/7/release-notes-20/#service-catalog-deprecation-update).
```

Comment:
```suggestion
In Kyma 2.0, we introduced a [new way to reach registered services and some Application Connectivity improvements](https://kyma-project.io/blog/2021/12/7/release-notes-20/#application-connectivity), which were related to the [Service Catalog deprecation](https://kyma-project.io/blog/2021/12/7/release-notes-20/#service-catalog-deprecation-update).
```

Reply:
This link actually leads to the deprecation note so maybe "deprecation" is a better word here?

Reply:
Applied you revised suggestion :)